crt_portal/crt_portal/settings.py

```diff
@@ -60,8 +61,12 @@
     'crt-portal-django-prod.app.cloud.gov',
     'crt-portal-django-stage.app.cloud.gov',
     'crt-portal-django-dev.app.cloud.gov',
+    '127.0.0.1',
```

Comment:
@LindsayYoung I think we want to remove this line because `127.0.0.1` is now set conditionally depending on `CIRCLE`.

Reply:
Thank you for catching that!!!!

Reply:
No problem!